src/native/_mutable.js

```diff
@@ -127,21 +127,18 @@ const translatePrivMDInput = (xorname, tag, secKey, nonce) => {
   let name = xorname;
   if (!Buffer.isBuffer(xorname)) {
     const b = new Buffer(xorname);
-    if (b.length != 32) throw Error("XOR Names _must be_ 32 bytes long.")
```

Comment:
Just for my own clarity, @hunterlester , we're removing these checks as it's handled by the underlying libs?

Reply:
Yeah, I'm removing them because our `XOR_NAME` ffi data type is already doing the checking and returns a helpful error.

Reply:
I was looking at this particular case last week as well, and since we are using `ref-array` for the `XOR_NAME`, `SYM_SECRET_KEY`, etc., then if we don't perform the length validation here the validation is made by `ref-array` rather than safe_app lib, but I don't think the error returned is that helpful, this is the type of error we get: `buffer length must be at least 32, got X`. So I'm not fully sure if we should stop returning our own message for this particular cases since you would get an error from `ref-array` but you wouldn't know which of the three args (`xorname`, `secKey` or `nonce`) is the one with the incorrect length.